docs/testing.md

```diff
@@ -20,13 +20,11 @@ For Minikube, ensure you build the image in the Minikube Docker context:
 ```
 
 For kind, you must manually `load` the `onos-config` image and the integration tests image 
-into the Kubernetes cluster each time the tests are modified:
+into the Kubernetes cluster each time the tests are modified. The `make kind` target will
```

Comment:
you might want to add that if you make changes to the simulators or cli or debug images you need to load those manually and provide the `kind load docker-image onosproject/<image>:<tag>` example

Reply:
@Andrea-Campanella the `make kind` target runs `kind load ...`

Reply:
@kuujo +1